static/templates/subscription_settings.handlebars

```diff
@@ -61,6 +61,14 @@
 
                     </li>
                     <li>
+                        <div id="sub_push_notifications_setting"
+                            class="sub_setting_checkbox sub_notification_setting {{#unless in_home_view}}muted-sub{{/unless}}">
+                            <input id="push-notifystream-{{stream_id}}" class="sub_setting_control" type="checkbox" tabindex="-1" {{#if push_notifications}}checked{{/if}} {{#unless in_home_view}}disabled="disabled"{{/unless}}/>
```

Comment:
We might not need some of the semi-duplicate IDs here?  An independent cleanup to consider while we're at it, at least.  Could make sense as a quick commit on the bottom of the branch.

Reply:
Was it the div ids that were unnecessary, and instead keep the input ids?